lib/ucfg.rb

```diff
@@ -48,6 +48,27 @@ def self.validate_recursively(config, schema, config_path)
           errors << "Property `#{(config_path + [key]).join('.')}` must be of type `#{schema['properties'][key]['type']}` (provided value `#{value}` of type `#{value_type(value)}`)"
         end
       end
+
+      if schema["properties"].key?(key) && schema["properties"][key].key?("min") && !schema["properties"][key].key?("max") && schema["properties"][key]["min"].is_a?(Numeric) && value.is_a?(Numeric)
```

Comment:
Why do we check for !schema["properties"][key].key?("max") also? Why is it required?

Reply:
Because there are 3 scenarios with different error messages:
- only min, no max
- only max, no min
- both max and min